<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1196</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualizador de Arquivos  .TIF</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>720</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>1134</width>
      <height>546</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="Layout_Principal">
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Eixo Y:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="intervalo_x_inicio"/>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="botao_limpar">
         <property name="text">
          <string>Limpar</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="intervalo_x_final"/>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="intervalo_y_final"/>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="botao_recorte">
         <property name="text">
          <string>Recortar</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="intervalo_y_inicio"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Eixo X:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_corte">
         <property name="text">
          <string>Defina o intervalo de corte:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Defina o nome do seu arquivo de corte:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nome_arquivo_corte"/>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_nomeArquivo">
       <property name="text">
        <string>Arquivo Selecionado :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGraphicsView" name="frame_exibicao_elevacao"/>
     </item>
     <item row="1" column="1">
      <widget class="QGraphicsView" name="frame_exibicao_gradiente"/>
     </item>
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="botao_abrir_arquivo">
         <property name="text">
          <string>Abrir</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="botao_mapa_rio">
         <property name="text">
          <string>Cidade do Rio</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_shape">
         <property name="text">
          <string>Shape: </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_mapa_rio">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>641</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Imagens Interface/mapa_rio.qrc">:/teste 2/mapa-rio-de-janeiro.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_regiao1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_regiao2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_regiao3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_regiao6">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>210</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_regiao9">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>210</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_regiao12">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>210</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_regiao5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_regiao4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_regiao7">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_regiao8">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>110</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_regiao10">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_regiao11">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>110</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1196</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivos">
    <property name="title">
     <string>Arquivos</string>
    </property>
   </widget>
   <addaction name="menuArquivos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Imagens Interface/mapa_rio.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>botao_limpar</sender>
   <signal>clicked()</signal>
   <receiver>intervalo_x_final</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>612</x>
     <y>594</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>563</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botao_abrir_arquivo</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ler_arquivo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botao_limpar</sender>
   <signal>clicked()</signal>
   <receiver>intervalo_y_inicio</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>612</x>
     <y>594</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>592</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botao_limpar</sender>
   <signal>clicked()</signal>
   <receiver>intervalo_y_final</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>612</x>
     <y>594</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>592</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botao_recorte</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>corta_tif()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>612</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>545</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botao_limpar</sender>
   <signal>clicked()</signal>
   <receiver>intervalo_x_inicio</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>612</x>
     <y>594</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>563</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botao_limpar</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>volta_tif()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>612</x>
     <y>594</y>
    </hint>
    <hint type="destinationlabel">
     <x>715</x>
     <y>545</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botao_limpar</sender>
   <signal>clicked()</signal>
   <receiver>nome_arquivo_corte</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>612</x>
     <y>594</y>
    </hint>
    <hint type="destinationlabel">
     <x>1149</x>
     <y>594</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botao_mapa_rio</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>mostra_mapa()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botao_regiao1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>botao_clicado_regiao()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botao_regiao2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>botao_clicado_regiao()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>30</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botao_regiao3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>botao_clicado_regiao()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botao_regiao4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>botao_clicado_regiao()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>619</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botao_regiao7</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>botao_clicado_regiao()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>710</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>774</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botao_regiao10</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>botao_clicado_regiao()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>924</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>972</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botao_regiao5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>botao_clicado_regiao()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>548</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>623</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botao_regiao6</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>botao_clicado_regiao()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>505</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>570</x>
     <y>629</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botao_regiao8</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>botao_clicado_regiao()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>664</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>704</x>
     <y>609</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botao_regiao9</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>botao_clicado_regiao()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>673</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>882</x>
     <y>627</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botao_regiao11</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>botao_clicado_regiao()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>845</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>1092</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botao_regiao12</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>botao_clicado_regiao()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>789</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>838</x>
     <y>645</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>abrir_arquivo()</slot>
  <slot>slot1()</slot>
  <slot>corta_tif()</slot>
  <slot>ler_arquivo()</slot>
  <slot>volta_tif()</slot>
  <slot>gera_gradiente()</slot>
  <slot>mostra_lista()</slot>
  <slot>ler_regiao_selecionada()</slot>
  <slot>slot2()</slot>
  <slot>mostra_mapinhas()</slot>
  <slot>ler_mapinha_selecionado()</slot>
  <slot>mostra_mapa()</slot>
  <slot>botao_clicado_regiao()</slot>
 </slots>
</ui>
